WordPressKitTests/Utilities/HTTPRequestBuilderTests.swift

```diff
@@ -140,6 +140,60 @@ class HTTPRequestBuilderTests: XCTestCase {
         )
     }
 
+    @available(iOS 16.0, *)
+    func testSetQueryWithDictionary() throws {
+        // The test data and assertions hre should be kept the same as the ones in `WordPressComRestApiTests.testQuery()`.
```

Comment:
> The test data and assertions here should be kept the same as the ones in `WordPressComRestApiTests.testQuery()`.

Have you considered extracting them? I noticed there is at least one additional value compared to `testQuery()` but I guess that could be addressed either with a dedicated test or by appending it here?

However, I realize that it might be overkill to got to that length for only two tests.